<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCtrlPrd</class>
 <widget class="QDialog" name="DlgCtrlPrd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Панель Управления ПРД</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>ВЫБОР</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textSelectPrd">
        <property name="text">
         <string>ПРД</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboSelectPrd">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxVisibleItems">
         <number>16</number>
        </property>
        <item>
         <property name="text">
          <string>ВСЕ</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textSelectModule">
        <property name="text">
         <string>УМИ</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboSelectModule">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxVisibleItems">
         <number>16</number>
        </property>
        <item>
         <property name="text">
          <string>ВСЕ</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="spacerSelect">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>УПРАВЛЕНИЕ</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupControlModule">
        <property name="title">
         <string>ИП УМИ</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushModuleOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushModuleOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupControlModule380">
        <property name="title">
         <string>380В УМИ</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushModule380On">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushModule380Off">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupControlPumi">
        <property name="title">
         <string>ПУМ</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushPumi1">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>I</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushPumi2">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>II</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="groupControlReset">
        <property name="title">
         <string>Сброс аварии</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <spacer name="spacerReset1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushReset">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>Сброс</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerReset2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupControlPower">
        <property name="title">
         <string>ПИТ ПРД</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushPowerOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushPowerOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupControlHotRes">
        <property name="title">
         <string>Горячий резерв ПУМ</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushHotResOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushHotResOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupControlSVCh">
        <property name="title">
         <string>СВЧ УМИ</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushSVChOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushSVChOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="spacerLine2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <spacer name="spacerLine3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacerClose1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerClose2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>190</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>comboSelectPrd</tabstop>
  <tabstop>comboSelectModule</tabstop>
  <tabstop>pushPowerOn</tabstop>
  <tabstop>pushPowerOff</tabstop>
  <tabstop>pushPumi1</tabstop>
  <tabstop>pushPumi2</tabstop>
  <tabstop>pushHotResOn</tabstop>
  <tabstop>pushHotResOff</tabstop>
  <tabstop>pushModule380On</tabstop>
  <tabstop>pushModule380Off</tabstop>
  <tabstop>pushModuleOn</tabstop>
  <tabstop>pushModuleOff</tabstop>
  <tabstop>pushSVChOn</tabstop>
  <tabstop>pushSVChOff</tabstop>
  <tabstop>pushReset</tabstop>
  <tabstop>pushClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
